docs/04_User_Guides/autofix_js_css_resources.md

```diff
@@ -0,0 +1,37 @@
+
+
+# How to apply autoformatting frontend resources
```

Comment:
The h1 title is added automatically with the name of the file. Can you please add a short description of what this guide brings to the user ?

Also you can add `[TOC]` alone on a line and it will create a table of contents.

Reply:
sure :+1:  didn't know about that features